internal/config/config.go

```diff
@@ -0,0 +1,28 @@
+package config
+
+import (
+	"strings"
+	"time"
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Configuration for this example app
+type Configuration struct {
+	ServicePort int `envconfig:"SERVICE_PORT" required:"false" default:"8082"`
+
+	// DB config
+	DBHost         string        `envconfig:"DB_HOST" required:"false" default:"127.0.0.1"`
+	DBUser         string        `envconfig:"DB_USER" required:"false" default:"root"`
+	DBPass         string        `envconfig:"DB_PASS" required:"false" default:"root" redact:"true"`

```

Comment:
https://github.com/vlam1/acceptance_test_example/blob/master/config/config.go

I got this from the example Vinh posted on the slack I thought we might have to stop the password from being displayed? I am not really sure thought it might be required that's why I included it

Reply:
I don't think you need it.